Discord Tools/BotList.md

```diff
@@ -1,8 +1,10 @@
 # Bot list
-1. ProBot - Reaction Roles, embed messages, metrics
-2. Hydra - Music
-3. Tickets - Support, Tickets
-4. Pheonix Rising Bot - Role prefixes
-5. SMP Bot - Custom-made staus bot, ask `Haxux#4306` for information
-6. CarlBot - Moderation, embed messages
-7. Poll Bot - Polls
+1. ProBot - Reaction Roles, embed messages, metrics                         Designation: 2
+2. Hydra - Music                                                            Designation: 4
+3. Tickets - Support, Tickets                                               Designation: 6
+4. Pheonix Rising Bot - Role prefixes                                       Designation: 3
+5. SMP Bot - Custom-made staus bot, ask `Haxux#4306` for information        Designation: 5
+6. CarlBot - Moderation, embed messages                                     Designation: 1
+7. Poll Bot - Polls                                                         Designation: 8
+8. Nonezero - Suggestion Bot                                                Designation: 9
+#### Designations are ordered in a way that the first one (1) is the most important, (9) being the least.

```

Comment:
Designation explanation. Possible revision: 1 to 6 are ordered in the way of importance, 7 to 9 are ordered in order of entrance